src/components/ec-multiple-values-selection/ec-multiple-values-selection.vue

```diff
@@ -220,6 +225,11 @@ function isItemChecked(item) {
 <style>
 @import '../../styles/tools/scrollbars';
 
+:root,
```

Comment:
what is the goal of the defining this custom variable in :root element? We don't use it anywhere else except this component.

Reply:
I can totally remove, I mostly tried to followed a pattern. I see a lot this lately without obvious reason but if you also find it weird will hard code instead.